docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,15 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
-first range of the table (i.e. containing the key `/<tableID>`). When
-a node receives a schema change operation such as `CREATE INDEX` it
+first range of the table (i.e. containing the key `/<tableID>`), known
+as the "table leader". Each node discovers via gossip the list of
+all configured tables, and starts a goroutine to execute schema
+changes on each table it is a leader for. The goroutine quits
+as soon as it discovers the node has lost its leader lease.
+It is critical, that the goroutine chunk its processing into small
+enough chunks, to relinquish leadership in a timely manner.
```

Comment:
I believe two nodes can perform the same schema change in parallel. The problem scenario is when one finishes and moves on to the next schema change. That is certainly going to be problematic.